esp/esp/program/modules/handlers/studentclassregmodule.py

```diff
@@ -330,24 +330,8 @@ def addclass_logic(self, request, tl, one, two, module, extra, prog):
         """ Pre-register the student for the class section in POST['section_id'].
             Return True if there are no errors.
         """
-        reg_perm = 'Student/Classes'
         scrmi = self.program.studentclassregmoduleinfo
 
-        if 'prereg_verb' in request.POST:
-            proposed_verb = "V/Flags/Registration/%s" % request.POST['prereg_verb']
-            if scrmi.use_priority:
-                available_verbs = ["%s/%d" % (scrmi.signup_verb.get_uri(), x) for x in xrange(1, scrmi.priority_limit+1)]
```

Comment:
Whoa, I was going to ask if this was ever used, and apparently `get_uri()` doesn't even exist anymore, so I guess not!